taxonomies/labels.txt

```diff
@@ -19252,6 +19252,27 @@ nl:Nutriscore E
 xx:Nutriscore E
 label_categories:en: en:Nutrition Grades, en:Health
 
+fr:Nutri-Score A et B
```

Comment:
ok

Reply:
done @stephanegigandet

Reply:
I meant for all of them :) also with the identical xx:Nutriscore A/B so that it matches in all languages, and is shown in all languages without the en: prefix.